### PR TITLE
Test auto-cancel of previous build in PR.

### PR DESCRIPTION
**Pull request by Yashwants19**
This PR is just to test auto-cancellation of previous build/run.

### CODE REVIEW

File: .github/workflows/cancel.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Get all workflow ids and set to env variable
+        run: echo ::set-env name=WORKFLOW_IDS_TO_CANCEL::$(curl https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/workflows -s | jq -r '.workflows | map(.id|tostring) | join(",")')
+
+      - uses: styfle/cancel-workflow-action@0.4.0
+        with:
+          workflow_id: ${{ env.WORKFLOW_IDS_TO_CANCEL }}
+          access_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (zoq) [inline .github/workflows/cancel.yml:15]:
```suggestion
          access_token: ${{ secrets.MLPACK_GITHUB_ACTIONS }}
```

Token created, maybe that works?

Comment (Yashwants19) [reply .github/workflows/cancel.yml]: This is showing some error.

Comment (Yashwants19) [reply .github/workflows/cancel.yml]: Have you install the app.

Comment (Yashwants19) [reply .github/workflows/cancel.yml]: I guess this is passing nothing.

**Pull request closed without merging**